migrations/2023-12-22-040137_make-mixed-sorting-directions-work-with-tuple-comparison/up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE FUNCTION reverse_timestamp_sort (t timestamp with time zone)
```

Comment:
since timestamptz is a 8-byte column but numeric is a variable-size column it might be good if this also returned of 8-byte size. for example bigint (`(-extract(epoch from t) * 1000)::bigint`)?

Reply:
done